.flutter-plugins

```diff
@@ -0,0 +1,15 @@
+# This is a generated file; do not edit or check into version control.
```

Comment:
Unstage this file, i think i changed our .gitignore on master to take care of this but you can add this locally and it will just be fixed when we merge